src/components/Layout/Layout.tsx

```diff
@@ -70,7 +76,7 @@ const Layout = ({
                   isAuthenticated={isAuthenticated}
                   activePage={activePage}
                 />
-                {showSearch && <SearchForm />}
+                {showSearch && searchForm}
```

Comment:
Why not update `searchForm` to be `showSearch || searchFormWithRefineResults || <SearchForm />`? There are now two logic checks in separate locations to render `<SearchForm />`.

Reply:
Ok! I updated it so the search page passes aggregations instead of a SearchForm to Layout and the search form decides what happens based on that prop.